### ISSUE TITLE
Ecommerce SmartContract

### ISSUE THREAD

**Issue by Satyam5665** [Vikash-8090-Yadav/Solidity-Pathshala#164]
Hello, I want to add a Smart-Contract for an Ecommerce Site. That will have buy, delivery, register-Products function that will ensure that only after the delivery of the Product is successful, money is deducted from the buyer account and sent to seller account.

**Comment by Vikash-8090-Yadav** [comment]
Yup , now u can start working on this . @Satyam5665

**Comment by Vikash-8090-Yadav** [comment]
Are u still working on this issue ?

**Comment by Debopom-Banerjee** [comment]
Hello, @Vikash-8090-Yadav as I could see there is no response from the assignee for a long time. Can I work on this issue?

**Comment by Vikash-8090-Yadav** [comment]
Don't forget to introduce yourself in the community !

**Comment by Debopom-Banerjee** [comment]
Yeah I have already introduced myself in the introduction channel in SWOC Discord server ! If there is any dedicated channel for Solidity-Pathsala can you please join me in.
Thanks

**Comment by Vikash-8090-Yadav** [comment]
@Debopom-Banerjee introduce yourself  here !
https://github.com/Vikash-8090-Yadav/Solidity-Pathshala/discussions/175

**Comment by Vikash-8090-Yadav** [comment]
And please give me your discord id also

**Comment by Debopom-Banerjee** [comment]
Hello Myself Debopom Banerjee, CSE undergrad from West Bengal, India. I am a full stack blockchain developer and also a MERN stack Developer. I am an Open-source enthusiast and also learning DSA in recent times.
Discord id : Debopom Banerjee#4862

**Comment by Vikash-8090-Yadav** [comment]
Click in this ->  https://github.com/Vikash-8090-Yadav/Solidity-Pathshala/discussions/175

**Comment by Debopom-Banerjee** [comment]
ohh I am sorry didn't noticed that
